keyboards/handwired/evk/v1_3/info.json

```diff
@@ -0,0 +1,629 @@
+{
```

Comment:
Please replace the contents of this file with the following:

```json
{
    "keyboard_name": "Ergonomic Vertical Keyboard (EVK) version 1.3 with Teensy 2.0",
    "url": "https://github.com/YangPiCui/ErgonomicVerticalKeyboard",
    "maintainer": "YangPiCui",
    "width": 22.75,
    "height": 7.5,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"label":"k0A", "x":0, "y":0.5, "w":1.25, "h":1.25},
                {"label":"k0B", "x":1.25, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0C", "x":2.5, "y":0, "w":1.25, "h":1.25},
                {"label":"k0D", "x":3.75, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0E", "x":5, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0F", "x":6.25, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0G", "x":8.75, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0H", "x":10, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0I", "x":11.25, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0J", "x":12.5, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0K", "x":15.25, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0L", "x":16.5, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0M", "x":17.75, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0N", "x":19, "y":0, "w":1.25, "h":1.25},
                {"label":"k0O", "x":20.25, "y":0.25, "w":1.25, "h":1.25},
                {"label":"k0P", "x":21.5, "y":0.5, "w":1.25, "h":1.25},
                {"label":"k1A", "x":0, "y":1.75, "w":1.25, "h":1.25},
                {"label":"k1B", "x":1.25, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1C", "x":2.5, "y":1.25, "w":1.25, "h":1.25},
                {"label":"k1D", "x":3.75, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1E", "x":5, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1F", "x":6.25, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1G", "x":8.75, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1H", "x":10, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1I", "x":11.25, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1J", "x":12.5, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1K", "x":15.25, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1L", "x":16.5, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1M", "x":17.75, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1N", "x":19, "y":1.25, "w":1.25, "h":1.25},
                {"label":"k1O", "x":20.25, "y":1.5, "w":1.25, "h":1.25},
                {"label":"k1P", "x":21.5, "y":1.75, "w":1.25, "h":1.25},
                {"label":"k2A", "x":0, "y":3, "w":1.25, "h":1.25},
                {"label":"k2B", "x":1.25, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2C", "x":2.5, "y":2.5, "w":1.25, "h":1.25},
                {"label":"k2D", "x":3.75, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2E", "x":5, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2F", "x":6.25, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2G", "x":8.75, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2H", "x":10, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2I", "x":11.25, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2J", "x":12.5, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2K", "x":15.25, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2L", "x":16.5, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2M", "x":17.75, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2N", "x":19, "y":2.5, "w":1.25, "h":1.25},
                {"label":"k2O", "x":20.25, "y":2.75, "w":1.25, "h":1.25},
                {"label":"k2P", "x":21.5, "y":3, "w":1.25, "h":1.25},
                {"label":"k3A", "x":0, "y":4.25, "w":1.25, "h":1.25},
                {"label":"k3B", "x":1.25, "y":4, "w":1.25, "h":1.25},
                {"label":"k3C", "x":2.5, "y":3.75, "w":1.25, "h":1.25},
                {"label":"k3D", "x":3.75, "y":4, "w":1.25, "h":1.25},
                {"label":"k3E", "x":5, "y":4, "w":1.25, "h":1.25},
                {"label":"k3F", "x":6.75, "y":4.5, "w":1.25, "h":1.25},
                {"label":"k3G", "x":8.75, "y":4, "w":1.25, "h":1.25},
                {"label":"k3H", "x":10, "y":4, "w":1.25, "h":1.25},
                {"label":"k3I", "x":11.25, "y":4, "w":1.25, "h":1.25},
                {"label":"k3J", "x":12.5, "y":4, "w":1.25, "h":1.25},
                {"label":"k3K", "x":14.75, "y":4.5, "w":1.25, "h":1.25},
                {"label":"k3L", "x":16.5, "y":4, "w":1.25, "h":1.25},
                {"label":"k3M", "x":17.75, "y":4, "w":1.25, "h":1.25},
                {"label":"k3N", "x":19, "y":3.75, "w":1.25, "h":1.25},
                {"label":"k3O", "x":20.25, "y":4, "w":1.25, "h":1.25},
                {"label":"k3P", "x":21.5, "y":4.25, "w":1.25, "h":1.25},
                {"label":"k4A", "x":0, "y":5.5, "w":1.25, "h":1.25},
                {"label":"k4B", "x":1.25, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4C", "x":2.5, "y":5, "w":1.25, "h":1.25},
                {"label":"k4D", "x":3.75, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4E", "x":5.5, "y":5.75, "w":1.25, "h":1.25},
                {"label":"k4F", "x":6.75, "y":5.75, "w":1.25, "h":1.25},
                {"label":"k4G", "x":8.75, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4H", "x":10, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4I", "x":11.25, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4J", "x":12.5, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4K", "x":14.75, "y":5.75, "w":1.25, "h":1.25},
                {"label":"k4L", "x":16, "y":5.75, "w":1.25, "h":1.25},
                {"label":"k4M", "x":17.75, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4N", "x":19, "y":5, "w":1.25, "h":1.25},
                {"label":"k4O", "x":20.25, "y":5.25, "w":1.25, "h":1.25},
                {"label":"k4P", "x":21.5, "y":5.5, "w":1.25, "h":1.25},
                {"label":"k5B", "x":1.25, "y":6.5, "w":1.25, "h":1.25},
                {"label":"k5C", "x":2.5, "y":6.25, "w":1.25, "h":1.25},
                {"label":"k5G", "x":8.75, "y":6.5, "w":1.25, "h":1.25},
                {"label":"k5H", "x":10, "y":6.5, "w":1.25, "h":1.25},
                {"label":"k5I", "x":11.25, "y":6.5, "w":1.25, "h":1.25},
                {"label":"k5J", "x":12.5, "y":6.5, "w":1.25, "h":1.25},
                {"label":"k5N", "x":19, "y":6.25, "w":1.25, "h":1.25},
                {"label":"k5O", "x":20.25, "y":6.5, "w":1.25, "h":1.25}
            ]
        }
    }
}
```

- changes the `maintainer` value, which is intended to be your GitHub username
- corrects the object order, which is currently incorrect and would lead to broken QMK Configurator support if merged as-is

Reply:
Thank you. I have changed all names to my Github username, YangPiCui.

Reply:
I have also updated info.json with the contents above.